admin/client/src/boot/BootInjector.js

```diff
@@ -0,0 +1,38 @@
+import injector from 'lib/Injector';
+import TextField from 'components/TextField/TextField';
+import HiddenField from 'components/HiddenField/HiddenField';
+import CheckboxField from 'components/CheckboxField/CheckboxField';
+import CheckboxSetField from 'components/CheckboxSetField/CheckboxSetField';
+import OptionsetField from 'components/OptionsetField/OptionsetField';
+import GridField from 'components/GridField/GridField';
+import SingleSelectField from 'components/SingleSelectField/SingleSelectField';
+import PopoverField from 'components/PopoverField/PopoverField';
+import HeaderField from 'components/HeaderField/HeaderField';
+import LiteralField from 'components/LiteralField/LiteralField';
+import HtmlReadonlyField from 'components/HtmlReadonlyField/HtmlReadonlyField';
+import CompositeField from 'components/CompositeField/CompositeField';
+import Tabs from 'components/Tabs/Tabs';
+import TabItem from 'components/Tabs/TabItem';
+import FormAction from 'components/FormAction/FormAction';
+
+class BootInjector {
```

Comment:
Done, added a comment for removing when dependency injection is in and left a comment that this was mainly to remove clutter from `boot/index.js`